css/_variables.scss

```diff
@@ -269,3 +269,9 @@ $chromeExtensionBannerTop: 80px;
 $chromeExtensionBannerRight: 16px;
 $chromeExtensionBannerTopInMeeting: 10px;
 $chromeExtensionBannerRightInMeeeting: 10px;
+
+/**
+* media type thresholds
+*/
+$smallScreen: 700px;
```

Comment:
We have to decide what are the right thresholds for that, I think this 2 thresholds for now are ok. Maybe in the future we want to add extra thresholds for ultra wide screens, or things like that.